eng/Version.Details.xml

```diff
@@ -664,9 +664,9 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>e77011b31a3e5c47d931248a64b47f9b2d47853d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Testing.Platform" Version="1.5.0-preview.24604.7">
+    <Dependency Name="Microsoft.Testing.Platform" Version="1.5.0-preview.24610.1">
```

Comment:
I think this repo will need to be added to the VMR.  To do this, it needs an entry added to https://github.com/dotnet/sdk/blob/main/src/VirtualMonoRepo/source-mappings.json.  Additionally since this is going to be required for source-build.  This dependency will need the source-build metadata added. 

`<SourceBuild RepoName="testfx" ManagedOnly="true" />`

Reply:
This may not be required now with this PR but it sounds like it will long term.